examples/scaling-criteo/01-Download-Convert.ipynb

```diff
@@ -1,304 +1,304 @@
 {
```

Comment:
<p>what's a practical value for XXX?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>